plugwise/helper.py

```diff
@@ -969,8 +1015,9 @@ def _scan_thermostats(self) -> None:
                     {"master": None, "master_prio": 0, "slaves": set()}
                 )
 
-            for appliance_id, appliance_details in self._appl_data.items():
-
+            for item in self._appl_data:
```

Comment:
Just asking out loud: the old way wasn't allowed because of typing?

Reply:
Correct, because self._appl_data is now a TypedDict, and the details are in a separate, named dict, see `constants.py`.